src/Products/Viewer/Controllers/ViewerApiController.cs

```diff
@@ -36,8 +36,8 @@ public class ViewerApiController : ApiController
         protected static readonly ConcurrentDictionary<string, object> KeyLockerMap = new ConcurrentDictionary<string, object>();
 
         private static readonly Common.Config.GlobalConfiguration globalConfiguration = new Common.Config.GlobalConfiguration();
-
-        private static readonly string cachePath = Path.Combine(globalConfiguration.Viewer.GetFilesDirectory(), globalConfiguration.Viewer.GetCacheFolderName());
+        
+        private IFileWrapper FileWrapper;
```

Comment:
Codacy found an issue: [Make 'FileWrapper' 'readonly'.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=6117781)